log_outgoing_requests/models.py

```diff
@@ -206,13 +206,23 @@ class OutgoingRequestsLogConfig(SingletonModel):
             "If 'Require content length' is not checked, this setting has no effect."
         ),
     )
+    reset_db_save_after = models.PositiveIntegerField(
+        _("Reset saving logs in database after"),
+        null=True,
+        blank=True,
+        help_text=_(
+            "If configured, after the config has been updated, reset the database logging "
+            "after the specified number of minutes. Note: this overrides the "
+            "LOG_OUTGOING_REQUESTS_RESET_DB_SAVE_AFTER environment variable."
+        ),
+    )
```

Comment:
I'd also add a min value validator to set at least 1 minute, 0 minutes would immediately reset it and makes it unclear in the code if the value is set to `0` or `None` with the soft-boolean check `if config.reset_db_save_after`

Adding the validator removes ambiguity